site/installing-weave.md

```diff
@@ -32,6 +32,8 @@ With Weave Net downloaded onto your VMs or hosts, you are ready to launch a Weav
 
 CoreOS users see [here](/guides/networking-docker-containers-with-weave-on-coreos/) for an example of installing Weave using cloud-config.
 
+Amazon AWS users may want to use AWSVPC?
```

Comment:
I was pondering whether this is the right place to link from - the others are Guides.